### ISSUE TITLE
Autoloading ramp options

### ISSUE THREAD

**Issue by justnorris** [Automattic/gutenberg-ramp#63]
Currently, when Ramp plugin is activated it autoloads `gutenberg_ramp_load_critera` on each and every page in the admin, including pages like the `/wp-admin/` root and `wp-admin/plugins.php`

As a result, this query happen on each admin page load:

```sql
SELECT autoload
FROM wp_options
WHERE option_name = 'gutenberg_ramp_load_critera'
```

I see 2 possible improvements we can make here:

1. Don't use autoloading for Gutenberg Ramp options - we only need the option in `edit.php`

2. Only check for Gutenberg in pages where it's likely that Gutenberg can appear, like `edit.php` and `posts-new.php`, etc.

Any thoughts on this?

**Comment by chetansatasiya** [comment]
@justnorris Norris! Can you look into my PR and let me know if any changes on that or any suggestion?

**Comment by maevelander** [comment]
Resolved in 1.1. Closing.
